pages/query-string.tsx

```diff
@@ -0,0 +1,68 @@
+import Typography from '@mui/material/Typography';
+import { GetStaticProps } from 'next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import React, { useState } from 'react';
+import { Namespace, useTranslation } from 'react-i18next';
+
+import Layout from '../components/Layout';
+import {
+  defaultParams,
+  QueryParams,
+} from '../components/queryString/QueryParamsForm';
+import QueryStringForm from '../components/queryString/QueryStringForm';
+import Toast, { ToastProps } from '../components/Toast';
+import useLocalState from '../hooks/useLocalState';
+import nextI18NextConfig from '../next-i18next.config.js';
+
+export default function QueryStringPage() {
+  const [queryString, setQueryString] = useLocalState<string | void>({
+    key: 'queryString',
+    defaultValue: '',
+  });
+  const [queryParams, setQueryParams] = useLocalState<QueryParams>({
+    key: 'queryParams',
+    defaultValue: defaultParams,
+  });
+
+  const [error, setError] = React.useState('');
+  const [toastOpen, setToastOpen] = useState<boolean>(false);
+  const [toastMessage, setToastMessage] = useState<string>('');
+  const [toastSeverity, setToastSeverity] =
+    useState<ToastProps['severity']>('success');
+  const { t } = useTranslation('queryString');
+  return (
+    <Layout title='Query String'>
```

Comment:
Please use the translation field from common here too, same one as the nav item.